extide/gradle/netbeans-gradle-tooling/src/main/groovy/org/netbeans/modules/gradle/tooling/NbProjectInfoBuilder.groovy

```diff
@@ -199,12 +199,15 @@ class NbProjectInfoBuilder {
         boolean hasJava = project.plugins.hasPlugin('java-base')
         boolean hasGroovy = project.plugins.hasPlugin('groovy-base')
         boolean hasScala = project.plugins.hasPlugin('scala-base')
+        boolean hasKotlin = project.plugins.hasPlugin('org.jetbrains.kotlin.android') ||
```

Comment:
I'd add these plugins to the detectPlugins (line 140) as well, so it other NetBeans modules can enrich the GradleProject Lookup when they are present.